src/clojure_tictactoe/game/game_rules.clj

```diff
@@ -23,7 +23,9 @@
   (some? (some #(apply = %) (potential-wins board))))
 
 (defn assess-winner [board]
-  (first (some #(when (apply = %) %) (potential-wins board))))
+  (-> #(when (apply = %) %)
```

Comment:
Sometimes anonymous functions can be pretty nice. But sometimes, they can make code a little more difficult to read. This is one of those times lol. I would name the arg in a function here instead of using an anonymous function.

Reply:
✅